### PR TITLE
I1 seir model

### PR DESCRIPTION
**Pull request by FarmHJ**
Addresses #1. Creates a class for SEIDR model. Unit tests, documentations and vignettes are included.

### CODE REVIEW

File: R/SEIRD.R [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+      2:length(output$D)] - output$D[1:(length(output$D) - 1)]
+
+    colnames(output) <- c("time", object@output_names)
+
+    # Create long format of output
+    output <- melt(output, id.vars = "time")
+    names(output) <- c("time", "compartment", "value")
+
+    # Added for consistency of output format across models
+    output$age_group <- rep("0-150", length(output$time))
```
Comment (oscardm20994) [inline R/SEIRD.R]: We should ensure the long format of the output is consistent with the SEIRAge model, this looks good but we've done it slightly differently. both would probably work well.

**Review by oscardm20994 — COMMENTED** [review]
This is looking really good, @solveigvandervegt and @FarmHJ. Just need to make sure there is consistency across this and the age model in outputs etc. Other than that looks in really good shape.

File: R/SEIRD.R (hunk 1 above)
Comment (solveigvandervegt) [reply R/SEIRD.R]: Thanks Oscar, I will change ours to match yours!

**Comment by solveigvandervegt** [comment]
@ben18785 I think I made all the changes that we talked about. Let me know if I missed anything!

**Review by ben18785 — APPROVED** [review]
Great work guys! I'm happy to merge now.

**Pull request merged**